interface/Cargo.toml

```diff
@@ -1,11 +1,13 @@
 [package]
 name = "solana-system-interface"
-version = "0.0.1"
+version = "1.0.0"
 description = "Instructions and constructors for the System program"
+authors = ["Anza Maintainers <maintainers@anza.xyz>"]
 repository = "https://github.com/solana-program/system"
+homepage = "https://anza.xyz/"
+license = "Apache-2.0"
 edition = "2021"
```

Comment:
nit: doesn't need to be done in this PR, but how about defining all these at the workspace level and using that, ie:

```
authors = { workspace = true }
repository = { workspace = true }
homepage = { workspace = true }
license = { workspace = true }
edition = { workspace = true }
```

Reply:
Good call. I will add them here.